components.json

```diff
@@ -0,0 +1,49 @@
+{
+  "sgov-service": {
+    "name": "SGoV Service",
+    "url": "https://kbss.felk.cvut.cz/sgov-server",
+    "meta": {}
+  },
+  "db": {
+    "name": "A graph database",
+    "url": "https://example.com/sparql",
+    "meta": {}
+  },
+  "auth": {
+    "name": "OIDC Authentication Service",
+    "url": "https://kbss.felk.cvut.cz:10808/auth/realms/kodi-dev",
+    "meta": {}
+  },
+  "ontographer": {
+    "name": "ontoGrapher",
+    "url": "https://slovník.gov.cz/modelujeme/v-nástroji/ontographer",
+    "meta": {
+      "workspacePath": "?workspace=%WORKSPACE_IRI%"
+    }
+  },
+  "termit-backend": {
+    "name": "TermIT backend",
+    "url": "https://example.com/termit-backend",
+    "meta": {}
+  },
+  "termit": {
+    "name": "TermIT",
+    "url": "https://slovník.gov.cz/modelujeme/v-nástroji/termit",
+    "meta": {
+      "workspacePath": "#vocabularies?workspace=%WORKSPACE_IRI%"
+    }
+  },
+  "mission-control": {
+    "name": "Mission Control",
+    "url": "https://example.com/mission-control",
+    "meta": {}
+  },
+  "issue-tracker": {
+    "name": "Issue Tracker",
+    "url": "https://github.com/opendata-mvcr/sgov-assembly-line/issues",
+    "meta": {
+      "newBug": "https://github.com/opendata-mvcr/sgov-assembly-line/issues/new?labels=bug&template=po-adavek-na-opravu.md&title=",
+      "newFeature": "https://github.com/opendata-mvcr/sgov-assembly-line/issues/new?labels=enhancement&template=po-adavek-na-novou-funkcionalitu.md&title="
+    }
+  }
+}
```

Comment:
This is a sample configuration file, it is here only for reference purposes. Eventually a similar file will likely be hosted in a repository with the master docker-compose script.